custom-checks/checkstyle/src/test/resources/checkstyle/noEmptyLineSeparatorCheck/EmptyLineAfterInvalidDefaultCaseWithoutBraces.java

```diff
@@ -0,0 +1,29 @@
+public class EmptyLineAfterInvalidDefaultCaseWithoutBraces {
+
+    // empty line after class is allowed
+
+    public void method() {
+        switch (channelId) {
+            case CHANNEL_WIND_DIRECTION:
+                return new DecimalType(windDirection);
+            case CHANNEL_AVG_WIND_SPEED:
```

Comment:
Maybe add a newline here, I believe this is not the focus point of this test, is it?

Reply:
I added a new line there and checked the test.